pkg/asset/installconfig/baremetal/baremetal.go

```diff
@@ -90,6 +90,7 @@ func Platform() (*baremetal.Platform, error) {
 			fmt.Printf("invalid host - please try again")
 			continue
 		}
+		host.Role = hostRole
```

Comment:
We could do something like this instead.... I don't know when the ecosystem is going to all get on the same page, TBH. The terms are so ingrained in the product maybe it's better if we don't continue?

```suggestion
		if hostRole == "control plane" {
			host.Role = "master"
		} else {
			host.Role = hostRole
		}
```